web/layouts/404.html

```diff
@@ -0,0 +1,21 @@
+{{ define "main" -}}
+
+<h2>Page not found!</h2>
+
+<p>We are sorry, but this page does not exist.
+<script>
+    if (document.referrer.search("/master") > 0) {
+        document.write("This page has been <a href=\"")
+        document.write(document.referrer.replace("/master", "/nightly"))
+        document.write("\">moved</a>.")
+    }
+</script>
+</p>
```

Comment:
This is a custom javascript that makes the switch easier for users who clicks on our links containing `/master`.